controllers/auth_config_controller.go

```diff
@@ -552,16 +643,50 @@ func (r *AuthConfigReconciler) translateAuthConfig(ctx context.Context, authConf
 	// denyWith
 	if responseConfig := authConfig.Spec.Response; responseConfig != nil {
 		if denyWith := responseConfig.Unauthenticated; denyWith != nil {
-			translatedAuthConfig.Unauthenticated = buildAuthorinoDenyWithValues(denyWith)
+			value, err := buildAuthorinoDenyWithValues(denyWith)
+			if err != nil {
+				return nil, err
+			}
+			translatedAuthConfig.Unauthenticated = value
 		}
 		if denyWith := responseConfig.Unauthorized; denyWith != nil {
-			translatedAuthConfig.Unauthorized = buildAuthorinoDenyWithValues(denyWith)
+			value, err := buildAuthorinoDenyWithValues(denyWith)
+			if err != nil {
+				return nil, err
+			}
+			translatedAuthConfig.Unauthorized = value
 		}
 	}
 
 	return translatedAuthConfig, nil
 }
 
+func stringValueFrom(user *api.ValueOrSelector) (expressions.Value, error) {
+	var strValue expressions.Value
+	var err error
+	if user.Expression != "" {
+		if strValue, err = cel.NewStringExpression(string(user.Expression)); err != nil {
+			return nil, err
+		}
+	} else {
+		strValue = &json.JSONValue{Static: user.Value, Pattern: user.Selector}
+	}
+	return strValue, nil
+}
+
+func valueFrom(user *api.ValueOrSelector) (expressions.Value, error) {
+	var strValue expressions.Value
+	var err error
+	if user.Expression != "" {
+		if strValue, err = cel.NewStringExpression(string(user.Expression)); err != nil {
+			return nil, err
+		}
+	} else {
+		strValue = &json.JSONValue{Static: user.Value, Pattern: user.Selector}
+	}
+	return strValue, nil
+}
```

Comment:
Is there any difference between these two funcs?

Reply:
Typo! This the latter should create a `NewExpression`! Tho I wonder if we should keep that one altogether... (by the current state of things, I'd say no!)